GliaWidgets/SecureConversations/SecureConversations.PendingInteraction.swift

```diff
@@ -29,6 +30,14 @@ extension SecureConversations {
                     hasPending || unreadCount > 0
                 }
                 .assign(to: &$hasPendingInteraction)
+
+            if self.pendingStatusCancellationToken == nil {
+                print("Unable to subscribe to pending status updates")
```

Comment:
should we use `log` instead `print`?

Reply:
It seems to me that this has to be a part of SDK configuration and affect SDK configuration completion result. So instead of adding logs I added error handling 047da9354611452871c37af00c4dd4ed50eb9c20.